src/components/organisms/JoinVenueModal/JoinVenueModal.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { FC, Fragment, useCallback } from "react";
+import { Modal } from "react-bootstrap";
+
+import { AnyVenue } from "types/venues";
+
+import { joinVenue } from "utils/url";
+
+import "./JoinVenueModal.scss";
+
+interface JoinVenueModalProps {
+  venueId?: string;
+  venue: AnyVenue;
+  show: boolean;
+  isLoggedIn: boolean;
+  onHide?: () => void;
+}
+
+export const JoinVenueModal: FC<JoinVenueModalProps> = ({
+  show,
+  venueId,
+  venue,
+  isLoggedIn,
+  onHide,
+}) => {
+  const handleJoinVenue = useCallback(() => {
+    joinVenue(venueId, venue.entrance);
+  }, [venue.entrance, venueId]);
+
+  const hideModal = useCallback(() => {
+    onHide && onHide();
+  }, [onHide]);
+
+  return (
+    <Modal show={show} onHide={hideModal}>
+      <Modal.Body>
+        <Fragment>
+          <div className="join-venue-title">
+            {isLoggedIn
+              ? `You are now attending ${venue.name}`
+              : "You need an account to join this party"}
+          </div>
```

Comment:
As per https://github.com/sparkletown/sparkle/pull/1134#discussion_r571795309

> "You are now attending :venueName" doesn't seem like the right sort of wording here.
> 
> @sparkletown/design @sparkletown/product Any thoughts?

Reply:
@denisdimitrov Would be good to check in on wording for this in standup tonight + also use whatever default terminology we decided on for 'party' loaded from `settings.ts`